echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/email/EmailNotificationService.groovy

```diff
@@ -49,8 +48,8 @@ class EmailNotificationService implements NotificationService {
   String from
 
   @Override
-  boolean supportsType(Notification.Type type) {
-    return type == TYPE
+  boolean supportsType(String type) {
+    return "EMAIL".equals(type.toUpperCase())
```

Comment:
i think it would at least be nice to have the string constants defined on the `Notification` class (or somewhere else)